docs/specification/current.md

```diff
@@ -555,16 +546,11 @@ The following example illustrates a delete action:
 }
 ```
 
-#### OverrideReason
-
-An **OverrideReason** is described by the following attributes. This specification does not prescribe a standard set of override reasons; implementers are encouraged to submit suggestions for standardization. 
+#### overrideReasons
 
-Field | Optionality | Type | Description
------ | ----- | ----- | --------
-`code` | REQUIRED | *string* | A code for this **OverrideReason**.
-`system` | OPTIONAL | *string* | A codesystem for this **OverrideReason** `code`.
-`display` | REQUIRED | *string* | A short, human-readable label to display for this override (e.g. the CDS Client might render this as the text on a button).
+**overrideReasons** is an array of **CDSHooksCoding** that captures a codified set of reasons an end user may select from as the rejection reason when rejecting the advice presented in the card. When using the coding object representing a reason, implementations are required to only respect the *code* property. However, they may consume other properties for a better end user experience, such as presenting a human readable text in the *display* property instead of the *code* itself to the end user. 
```

Comment:
```suggestion
**overrideReasons** is an array of **Coding** that captures a codified set of reasons an end user may select from as the rejection reason when rejecting the advice presented in the card. When using the coding object representing a reason, implementations are required to only respect the *code* property. However, they may consume other properties for a better end user experience, such as presenting a human readable text in the *display* property instead of the *code* itself to the end user. 
```